src/schemes/openIDConnect.ts

```diff
@@ -0,0 +1,273 @@
+import { AxiosResponse } from 'axios'
+import {
+  encodeQuery,
+  parseQuery,
+  normalizePath,
+  getResponseProp
+} from '../utils'
+import IdToken from '../inc/id-token'
+import ConfigurationDocument from '../inc/configuration-document'
+import type { SchemeCheck } from '../index'
+import Oauth2Scheme from './oauth2'
+
+const DEFAULTS = {
+  name: 'openIDConnect',
+  idToken: {
+    property: 'id_token',
+    maxAge: 60 * 60 * 24 * 30,
+    prefix: '_id_token.',
+    expirationPrefix: '_id_token_expiration.'
+  },
+  codeChallengeMethod: 'S256'
+}
+
+export default class OpenIDConnectScheme extends Oauth2Scheme<typeof DEFAULTS> {
```

Comment:
There is a new way to extend scheme options. Take a look at https://github.com/nuxt-community/auth-module/blob/dev/src/schemes/oauth2.ts#L39

Reply:
Thanks for the offer. I think I managed the upgrade path.